docs/Node Operators/best-security-practices.md

```diff
@@ -1,19 +1,23 @@
 ---
 layout: nodes.liquid
 date: Last Modified
-title: "Best Security and Operating Practices"
+title: "Security and Operating Best Practices"
 permalink: "docs/best-security-practices/"
 ---
 The following information provides a set of best security and operating practices that node operators need to utilize at a minimum to enhance the security and reliability of their infrastructure.
 
+## Provide Reliable Data
+
+{% include single-source/data-reliability-best-practices.md %}
```

Comment:
@dwightjl this includes is more geared towards developers and less toward node operators. The verbiage for node operators would be different as it would less about "protect your applications" and more about "provide reliable data". Perhaps something along the lines of:
```
> 🚧 Users applications are only as reliable as the data you provide
> 
> When users run a production application, there are risks associated with each data provider that you use.

To protect data consumers, apply the following guidelines:

- Be cautious when using data providers with only a single data source, which can be a single point of failure. Any error or omission in the provider's data might negatively impact your data and its consumers.
- Consider the incentives that malicious actors might have to manipulate the data from each data provider.
- For pricing data providers, consider the liquidity of the exchange where that pricing data is coming from. Exchanges and asset pairs with low liquidity and high volatility are vulnerable to price manipulation. If your node provides such pricing data, consumers' applications are equally vulnerable to manipulation.
```

Reply:
Affirmative. Applied to the latest commit.